src/middlewares/blacklistMiddleware.js

```diff
@@ -0,0 +1,33 @@
+import Redis from 'ioredis';
+import response from '../utils/response';
+
+const redis = new Redis(process.env.REDIS_URL)
+
+const checkBlacklist = async(req, res, next) => {
+    
+    const token = req.headers.authorization && req.headers.authorization.split(' ')[1]
+
+    if(!token){
+        return response(res, 401, 'error', {
+            message: "Token missing, you need a token to have access"
```

Comment:
let reference our message from the message object

Reply:
Changed to message object